pkg/config/config.go

```diff
@@ -1124,6 +1120,18 @@ func (c *Config) ActiveDestination() (uri, identity string, machine bool, err er
 		return d.URI, d.Identity, d.IsMachine, nil
 	case c.Engine.RemoteURI != "":
 		return c.Engine.RemoteURI, c.Engine.RemoteIdentity, false, nil
+	case connEnv != "":
```

Comment:
Handle both ENV variables the same way.  Either both in the switch statement or both 
as if blocks after the switch statement.  I think it is cleaner if they are both handled in the switch statement.